package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "utm-client",
+  "version": "1.0.0",
+  "description": "UTM attribution client",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint . --ext .js --cache"
```

May as well add a fix script to?
```suggestion
    "lint": "eslint . --ext .js --cache",
    "lint:fix": "eslint . --ext .js --fix"
```